src/rdkafka.h

```diff
@@ -1804,12 +1804,29 @@ rd_kafka_queue_t *rd_kafka_queue_get_main (rd_kafka_t *rk);
 RD_EXPORT
 rd_kafka_queue_t *rd_kafka_queue_get_consumer (rd_kafka_t *rk);
 
+/**
+ * @returns a reference to the librdkafka partition queue.
+ *
+ * Use rd_kafka_queue_destroy() to loose the reference.
+ *
```

Comment:
`@returns a reference to the partition's queue, or NULL if partition is invalid.`